2_observations.yml

```diff
@@ -134,3 +135,20 @@ targets:
       mgd_to_cms = 0.0438, cfs_to_cms = 0.0283,
       mg_to_cm = 4404.88, ft_to_m = 0.3048)
 
+ # reservoir surface area data
+ # you must download reservoir Monthly Shapefiles from here: 'http://umnlcc.cs.umn.edu/realsat/'
+ # unzip downloaded folder to 2_observations/in
+ # such that the structure becomes
+ # 2_observations/in/ID-Number
+      # in/573567/573567
+      # in/009130/009130
+  2_observations/out/reserviors_sf_directory:
+    command: read_monthly_surface_area_data(res_sf_dir = target_name)
```

Comment:
This function returns a list of directory names. That list (as an R object) does not get used in `combine_reaLSAT_reservoir_data`, which means there's a pipeline break between this target and the next one. This is an important connection to fix, and I will leave it to you two (Rasha and Hayley) to choose and implement a solution.